rng_guesser/number_guessing_game/src/main.rs

```diff
@@ -0,0 +1,59 @@
+use rand::thread_rng;
+use rand::Rng;
+
+use std::io;
+
+fn main() {
+    println!("Program Started...");
+
+    let start_txt = "
+██████╗     ███╗   ██╗     ██████╗              ██████╗ ██╗   ██╗███████╗███████╗███████╗███████╗██████╗ 
+██╔══██╗    ████╗  ██║    ██╔════╝             ██╔════╝ ██║   ██║██╔════╝██╔════╝██╔════╝██╔════╝██╔══██╗
+██████╔╝    ██╔██╗ ██║    ██║  ███╗            ██║  ███╗██║   ██║█████╗  ███████╗███████╗█████╗  ██████╔╝
+██╔══██╗    ██║╚██╗██║    ██║   ██║            ██║   ██║██║   ██║██╔══╝  ╚════██║╚════██║██╔══╝  ██╔══██╗
+██║  ██║    ██║ ╚████║    ╚██████╔╝            ╚██████╔╝╚██████╔╝███████╗███████║███████║███████╗██║  ██║
+╚═╝  ╚═╝    ╚═╝  ╚═══╝     ╚═════╝              ╚═════╝  ╚═════╝ ╚══════╝╚══════╝╚══════╝╚══════╝╚═╝  ╚═╝                                                                                                                                                                                                                                         
+    ";
+
+    println!("{start_txt}");
+    println!("Made by lucasodev - UNLICENSE");
+    println!(" ");
+    println!(" ");
+    
+    guessing();
+}
+
+fn guessing() {
+    let random_number = get_random_number();
+    
+    let mut guess_right: bool = false;
+
+    while guess_right == false {
+        let guess = get_user_input();
+        if guess == random_number {
+            println!("Yeah, it was right, the number was {random_number}!");
+            guess_right = true;
+        } else {
+            println!("Sorry, Your answer was wrong!");
+            continue;
+        }
+    }
+}
+
+fn get_random_number() -> i64 {
+    let _random_number: i64 = thread_rng().gen_range(0, 11);
```

Comment:
Unnecessary variable `_random_number` :

You could re-write this into:
```rust
fn get_random_number() -> i64 {
    thread_rng().gen_range(0, 11)
}
```

Reply:
Yeah